symfony/console/5.2/bin/console

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env php
+<?php
+
+use App\Kernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+if (!is_file(dirname(__DIR__).'/vendor/symfony/runtime/autoload.php')) {
+    throw new LogicException('Symfony Runtime is missing. Try running "composer require symfony/runtime".');
```

Comment:
Should we do the same in `index.php`?

Reply:
I think we can skip this in index.php: it will save some perf, and since the console is run when composer install/update runs also, the message will be displayed.